src/test/resources/configuration.properties

```diff
@@ -1,5 +1,6 @@
 #fillme
 #url=OCTANE_URL
 #sharedSpaceId=SHARED_SPACE_ID
+#workspaceId=WORKSPACE_ID
```

Comment:
will need to update the current ci, merging this will make it fail

Reply:
YES